### ISSUE TITLE
all-but-illegible text style needs fix for humans

### ISSUE THREAD

**Issue by TallTed** [w3c/vc-data-model#777]
Brought to eye by #775, the `<span class="color-text" style="color:#b4a7d6;">` and similar [here in the source](https://github.com/w3c/vc-data-model/blob/6c429cfc54ebfa6f5b74a95dd51d46eed7856b9e/index.html#L834-L843), and [here in the preview (describing figures 7 and 8)](https://w3c.github.io/vc-data-model/#basic-vp) is all-but-illegible in my browser (Chrome Version 90.0.4430.93 (Official Build) (x86_64) on macOS El Capitan (10.11.6)). I does't seem likely that it would be much more legible in any other browser on any other platform.

<img width="817" alt="illegible_text" src="https://user-images.githubusercontent.com/2559602/116598394-c5dcca80-a8f4-11eb-8de0-34aa945a791b.png">

Regrettably, this styling didn't go through W3 CSS, so it probably counts as an errata for W3 purposes.

I *am* surprised it passed muster for the Accessibility review!

**Comment by brentzundel** [comment]
We agree this needs fixing.

**Comment by kdenhartog** [comment]
Noting that the text coloing is intended to be aligned with the color coordination of the graph below it. Would we prefer to update this to the links of the definitions and include a paraentheses to indicate color association or drop the color association between the text and figure 8 all together?

**Comment by TallTed** [comment]
Tangentially, I would split the descriptions of Figure 7 and Figure 8 into separate paragraphs. (Yes, Fig 7 will be short.  That's OK.)

I would suggest a dotted-line in each color, surrounding the blocks of that color, extending to the right of the illustration, where the textual label for each collection should probably be in black.

With re-re-reading, I am thinking that the text should also be tweaked a bit, and either instead of or in addition to the "{first, second, third, fourth} graph", those graphs should be identified by the name used to label the current dashed-black-line blobs (i.e., "{Presentation, Credential, Credential Proof, Presentation Proof} Graph").  Without so doing, it can appear that the latter names are out of sync with the names used in the textual description -- which makes this already complex section even more difficult to comprehend.  I've taken a whack at this in #785.

**Comment by TzviyaSiegman** [comment]
I haven't looked at this document in a long time, but I did an accessibility review years ago. If the colors are conveying meaning, the meaning needs to be conveyed by more than just color for those who cannot perceive color. (I can perceive color, and I am unsure what is being conveyed by the color, and I find it hard to read.)

**Comment by chaals** [comment]
Yeah, I don't think this should have passed accessibility review - relying purely on colours to make the associations is a basic no-no. 

IMHO the images themselves would not pass accessibility review (that they are SVG is great, but that they are just a collection of path elements with no structure even for text is really bad).

Fixing this probably means using different styles for the diagram, with a more explicit legend.

I'll do a couple of hours of work to see how much improvement that would make...

**Comment by kdenhartog** [comment]
A second part of this issue was discovered after we addressed the first part. The details of the investigation and the PR can be found in issue #837 and PR #841 adding a comment here since they're directly related to follow up fixes that were meant to be addressed in this issue.
